src/app/shared/entity/external-machine-deployment.ts

```diff
@@ -74,6 +74,11 @@ export class ExternalMachineDeploymentPatch {
   spec?: ExternalMachineDeploymentSpecPatch;
 }
 
-class ExternalMachineDeploymentSpecPatch {
+export class ExternalMachineDeploymentSpecPatch {
   replicas?: number;
+  template?: {
+    versions: {
+      kubelet: 'string';
```

Comment:
```suggestion
      kubelet: string;
```

Should type be like this?